### ISSUE TITLE
Change how saved text is displayed

### ISSUE THREAD

**Issue by trife** [PhenoApps/Field-Book#85]
Currently relies on color - change to something that doesn't rely on a specific color change (e.g. black text on white background → white text on black background)

> For red/green color blind. Also: on a categorical trait, once you assign a category the text changes to red, which is nice but hard for me to see (I'm a bit colorblind, and it's especially hard to see in bright light).  If you move off that record and then go back, the assigned value now has a blue background, which is much easier to see.  it would be nice if the background changed like that as soon as you assigned the value.  I have had the experience of thinking I pressed the trait and going back to the record and finding that it was not assigned.
